book_review/app/controllers/book_ratings_controller.rb

```diff
@@ -0,0 +1,73 @@
+class BookRatingsController < ApplicationController
+  before_action :set_book_rating, only: [:show, :edit, :update, :destroy]
+
+  # GET /book_ratings
+  # GET /book_ratings.json
+  def index
```

Comment:
I think in project will not be present page with ALL book ratings, you can remove index method, route and views